src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
+    }
+    autocompletion += `>`;
+    // TODO(timvdlippe): once insertionPoints are implemented in analyzer, uncomment this code
```

Comment:
Should I just remove the code then? I primarily left it in to show the potential of this function, since without I feared this PR would be seen as unnecessary complexity.